[12042022 15:38:39] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 15:38:39] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 15:38:43] log /root/dionaea/src/signals.c:36-warning: sigterm_cb loop 0x7f1e6d3e9720 w 0x5588f747a230 revents 1024
[12042022 15:39:03] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 15:39:03] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 15:40:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:41:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:42:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:43:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:44:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:45:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:46:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:47:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:48:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:49:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:50:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:51:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:52:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:53:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:54:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:55:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:56:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:57:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:58:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:59:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:00:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:01:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:02:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:03:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:04:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:05:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:05:43] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:05:49] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:05:49] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:06:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:07:30] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[12042022 16:07:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:08:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:09:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:10:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:11:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:11:56] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:12:00] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:12:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:13:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:14:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:15:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:16:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:17:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:17:15] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:17:19] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:17:19] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:18:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:19:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:20:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:21:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:22:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:23:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:23:22] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:23:27] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:23:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:24:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:25:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:26:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:27:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:28:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:29:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:30:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:31:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:32:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:33:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:34:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:35:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:36:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:37:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:38:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:39:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:40:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:41:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:42:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:43:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:44:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:45:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:46:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:47:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:48:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[12042022 16:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[12042022 16:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:58:32] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[12042022 16:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:00:51] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[12042022 17:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:02:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:03:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 18:31:14] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 18:31:14] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 18:32:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:33:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:54] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:54] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:38:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:39:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:40:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:42:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:43:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:44:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:49:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:54:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:57:27] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 20:22:45] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 20:22:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 20:23:56] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:23:56] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:23:56] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:25:31] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:29:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:31:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:32:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:32:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:32:46] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[12042022 20:32:46] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[12042022 20:34:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:35:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:36:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:38:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:39:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:40:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:41:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:43:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:46:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:46:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:46:15] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:46:15] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:46:15] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:46:15] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:46:15] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:46:15] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:46:15] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:46:15] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 21:50:33] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 21:50:33] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 21:50:39] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 21:50:39] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 21:51:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:52:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:53:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:54:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:55:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:56:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:57:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:58:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 21:59:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:00:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:01:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:02:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:03:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:04:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:06:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:07:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:08:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:09:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:10:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:11:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:13:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:21:11] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:30:24] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:40:16] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:49:33] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 22:59:14] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:08:43] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:17:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:26:47] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:30:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:31:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:33:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:36:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:38:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:41:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:44:21] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:47:36] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:49:31] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:51:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:57:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 23:59:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:02:11] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:06:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:10:11] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:15:31] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:23:16] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:30:43] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:38:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:47:43] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 00:56:53] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:06:11] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:15:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:23:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:32:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:42:31] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:50:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:51:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:53:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:54:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:57:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:58:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 01:59:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:01:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:03:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:04:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:05:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:08:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:09:53] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:09:53] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[13042022 02:14:41] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[13042022 02:14:41] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[13042022 02:16:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:18:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:20:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:21:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:23:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:25:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:28:17] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:28:17] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:28:17] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:34:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:34:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:37:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:39:17] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:41:13] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:41:13] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:44:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:45:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:47:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:48:00] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[13042022 02:48:00] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[13042022 02:48:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:49:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:50:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:18] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[13042022 02:51:18] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[13042022 02:51:18] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 02:51:42] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:00:04] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:02:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:04:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:07:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:10:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:13:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:17:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:20:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:23:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:25:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:26:00] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[13042022 03:26:00] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[13042022 03:26:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:26:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:26:00] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[13042022 03:26:00] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[13042022 03:26:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:26:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:50:32] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[13042022 03:50:32] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
